.github/workflows/common.yml

```diff
@@ -11,28 +11,26 @@ on:
   # 'push' events are triggered when commits
   # are pushed to one of these branches
   push:
+    tags:
+      - "v*"
     branches:
       - main
       - devel
+      - patch
       - pre-release
```

Comment:
can we remove references to `pre-release`

Reply:
Should I actually just copy over the entire [admiralci/.github/workflows](https://github.com/pharmaverse/admiralci/tree/main/.github/workflows) directory? It looks like using the `common.yml` is out of date?

Reply:
Yeah that might be best

Reply:
@zdz2101 @bms63 i might be missing the point here and sorry if so, but wasn't the whole point of using `common.yml` that we don't ever need to copy the individual workflows over from admiralci? Tagging @bundfussr @thomas-neitmann here to comment just in case, as not sure i understand this suggested update

Reply:
I think he is just talking about what is in admiral workflow folder - which is common, cran status (leave that one alone) and template check 
![image](https://user-images.githubusercontent.com/10111024/230076427-488a14f1-b5d0-454b-baba-02c2a40b838d.png)

Reply:
i'm not sure what's the difference between `templates.yml` in admiral and `check-templates.yml` in admiralci? my point was more not to copy over all the admiralci workflows here, as they should live only there and then we reference them in each package ext through the `common.yml` - as per https://pharmaverse.github.io/admiralci/main/index.html#reuse-recommended

Reply:
Yes I definitely was a bit confused there too, I went ahead and added the `templates.yml` which is a `.yml` that triggers the  Check-Templates action after an review approves the PR. As for the `common.yml` I updated to remove the mention of `pre-release` , speaking of which, we need to update for `admiral` and `admiraldev`. I can go ahead and make issues for that too (good first PR)

Reply:
great, thanks Zelos!